listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -263,6 +263,23 @@ export default class RecentListens extends React.Component<
           return;
         }
       }
+      // The websocket message received contains the recording_msid as a top level key.
+      // Therefore, we need to shift it json.track_metadata.additional_info.
+      if (!("recording_msid" in json.track_metadata.additional_info)) {
+        if ("recording_msid" in json) {
+          json.track_metadata.additional_info.recording_msid =
+            json.recording_msid;
+          delete json.recording_msid;
+        } else {
+          // eslint-disable-next-line no-console
+          console.debug(
+            `Could not find recording_msid in following json: ${json}`
```

Comment:
This will result in `Could not find recording_msid in following json: [object Object]` in the console.
You can pass the json object as a second argument for it to be printed with correct formatting:

```suggestion
            "Could not find recording_msid in following json:", json
```

Reply:
done